src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -159,34 +159,36 @@
         </div>
     </div>
     <div id="right-options-panel" class="flex-window">
-        <div class="panel document-properties flex-window-head">
+        <div class="panel document-properties flex-window-head" id="document-properties-area">
```

Comment:
we already have nice selector for that: document-properties no need for new one.

Reply:
done